### PR TITLE
find outputs for ant-style builds

### PR DESCRIPTION
**Pull request by djjanse**
Hi Sam, we still use ant style android builds for Cordova (by using the ANDROID_BUILD=ant env variable). The plugin cannot find the output apk's then, because for ant builds they are in the `androidPath/bin` directory. Could you include this fix in your package?

### CODE REVIEW

**Comment by SamVerschueren** [comment]
Thanks for the PR. Would it make sense to just check `process.env.ANDROID_BUILD === 'ant'` and then switch the directory? Or is it possible to trigger an `ant` build via another way as well? Not sure what the best approach would be to be honest, so feedback is always welcome.

If we go for the path exists call, use [path-exists](https://github.com/sindresorhus/path-exists). `fs.exists[Sync]` is [deprecated](https://nodejs.org/api/fs.html#fs_fs_existssync_path).

**Comment by djjanse** [comment]
I made an extra option in the API `buildMethod`. This way it's more explicit to choose the build method. As an alternative the environment variable `ANDROID_BUILD`can be used, however the `buildMethod` option takes precedence.

File: index.js [hunk 1]
```diff
@@ -57,13 +58,20 @@ module.exports = function (options) {
...
 
 			// Build the platform
 			return cordova.build({platforms: ['android'], options: options});
 		}).then(function () {
-			var base = path.join(androidPath, 'build/outputs/apk');
+			var apkOutputPath = buildMethod === 'ant' ? 'bin' : 'build/outputs/apk';
+			var base = path.join(androidPath, apkOutputPath);
 			var cwd = process.env.PWD;
-
+
```
Comment (SamVerschueren) [inline index.js]: Newline is fine here, but remove the indentation.

File: readme.md [hunk 2]
```diff
@@ -125,6 +125,13 @@ Type: `string`
 
 Version of [`cordova-android`](https://github.com/apache/cordova-android) platform plugin.
 
+##### buildMethod
+
+Type: `string`
+Default: `gradle`
+
+Build method to use. Either `ant` or `gradle`. 
+
```
Comment (SamVerschueren) [inline readme.md:134]: Add an extra newline

File: readme.md [hunk 3]
```diff
@@ -125,6 +125,13 @@ Type: `string`
 
 Version of [`cordova-android`](https://github.com/apache/cordova-android) platform plugin.
 
+##### buildMethod
+
+Type: `string`
```
Comment (SamVerschueren) [inline readme.md]: Add `<br>` at the end of the line

**Comment by SamVerschueren** [comment]
Nice, looks good! Added some extra notes. Will merge after you fixed them :)

**Comment by djjanse** [comment]
Fixed the formatting. Thank for you comments!

File: index.js [hunk 4]
```diff
@@ -58,10 +59,17 @@ module.exports = function (options) {
 				options.push('--release');
 			}
 
+			if (buildMethod === 'ant'){
+				options.push('--ant');
+			}else{
```
Comment (SamVerschueren) [inline index.js]: `} else {`

**Comment by SamVerschueren** [comment]
Just one more. Need to add a linter and tests to catch this asap :).

**Comment by djjanse** [comment]
OK, found still one more :-). Yes, linting and tests would be nice. If I find the time, I'll be happy to help you with that.

**Comment by SamVerschueren** [comment]
@djjanse Thank you very much! Released as `1.5.1`.

I'm all open for PR on [linting](https://github.com/sindresorhus/xo), [testing](https://github.com/avajs/ava) and all other improvements you can think of. [gulp-cordova-plugin](https://github.com/SamVerschueren/gulp-cordova-plugin) has all of that so it might be useful as inspiration. I'm also going to drop support for node 0.10 and 0.12. It would clean up the code drastically with ES2015 features like fat-arrow functions.

**Pull request merged**